mapengine/worldwind/apk/src/main/java/mil/emp3/worldwind/MapInstance.java

```diff
@@ -263,6 +265,7 @@ protected void initMapEngine(Context context, AttributeSet attrs) {
         this.oMapViewController = new Emp3NavigationListener(this, ww);
         this.handler = new Handler(Looper.getMainLooper());
         this.wmsHash = new HashMap<>();
+        this.wmtsHash = new HashMap<>();
```

Comment:
Can't we have one mapServiceHash and add WMS and WMTS, all map service layers on it.